### PR TITLE
Repositories data

### PR DESCRIPTION
**Pull request by JuanitoFatas**
<kbd>[View Rendered README.md](https://github.com/jollygoodcode/data.rubyasia.com/blob/c9f01fe/README.md)</kbd>

<kbd>[View Data](https://github.com/jollygoodcode/data.rubyasia.com/commit/c9f01fe)</kbd>

### CODE REVIEW

File: app/repo.rb [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    SLICE_NUMBER = 300
+
+    private_constant :ROOT
+    private_constant :DATA_USERS_FOLDER
+    private_constant :DATA_REPOS_FOLDER
+    private_constant :SLICE_NUMBER
+
+    attr_reader :raw_region, :available_regions, :noramlized_regions, :regions
+
+    def determin_by_user_specified_region(user_specified_region)
```
Comment (winston) [inline app/repo.rb]: `determine`

File: app/repo.rb [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+      if user_specified_region == "all".freeze
+        noramlized_regions
+      else
+        if noramlized_regions.include?(user_specified_region)
```
Comment (winston) [inline app/repo.rb]: `normalized`

**Comment by winston** [comment]
So the criteria is:

>   "language:Ruby stars:>20 pushed:#{period}"

right?

Ruby repo with stars > 20 and last commit in the last 7 days?

If so, the data looks great! Am thinking if we should extend the period though.

**Comment by JuanitoFatas** [comment]
> Ruby repo with stars > 20 and last commit in the last 7 days?

yes. A week before yesterday.

**Pull request merged**